packages/hardhat-toolbox/package.json

```diff
@@ -0,0 +1,89 @@
+{
+  "name": "@nomicfoundation/hardhat-toolbox",
+  "private": true,
+  "version": "0.0.1",
+  "description": "",
+  "repository": "github:nomicfoundation/hardhat",
+  "homepage": "https://github.com/nomicfoundation/hardhat/tree/master/packages/hardhat-toolbox",
+  "author": "Nomic Foundation",
+  "contributors": [
+    "Nomic Foundation"
+  ],
+  "license": "MIT",
+  "main": "dist/src/index.js",
+  "types": "dist/src/index.d.ts",
+  "keywords": [
+    "ethereum",
+    "smart-contracts",
+    "hardhat",
+    "hardhat-plugin"
+  ],
+  "scripts": {
+    "lint": "yarn prettier --check && yarn eslint",
+    "lint:fix": "yarn prettier --write && yarn eslint --fix",
+    "eslint": "eslint 'src/**/*.ts' 'test/**/*.ts'",
+    "prettier": "prettier \"**/*.{js,md,json}\"",
+    "test": "mocha --recursive \"test/**/*.ts\" --exit",
+    "build": "tsc --build .",
+    "clean": "rimraf dist"
+  },
+  "files": [
+    "dist/src/",
+    "src/",
+    "LICENSE",
+    "README.md"
+  ],
+  "dependencies": {},
+  "devDependencies": {
+    "@ethersproject/abi": "^5.1.2",
+    "@ethersproject/providers": "^5.4.7",
+    "@nomicfoundation/hardhat-network-helpers": "*",
+    "@nomicfoundation/hardhat-chai-matchers": "*",
+    "@nomiclabs/hardhat-ethers": "^2.0.0",
+    "@nomiclabs/hardhat-etherscan": "^3.0.0",
+    "@typechain/ethers-v5": "^10.0.0",
+    "@typechain/hardhat": "^6.0.0",
+    "@types/chai": "^4.2.0",
+    "@types/mocha": "^9.1.0",
+    "@types/node": "^12.0.0",
+    "@typescript-eslint/eslint-plugin": "4.29.2",
+    "@typescript-eslint/parser": "4.29.2",
+    "chai": "^4.2.0",
+    "eslint": "^7.29.0",
+    "eslint-config-prettier": "8.3.0",
+    "eslint-plugin-import": "2.24.1",
+    "eslint-plugin-prettier": "3.4.0",
+    "ethers": "^5.0.0",
+    "hardhat": "^2.9.4",
+    "hardhat-gas-reporter": "^1.0.8",
+    "mocha": "^9.2.0",
+    "prettier": "2.4.1",
+    "rimraf": "^3.0.2",
+    "solidity-coverage": "^0.7.21",
+    "ts-node": "^8.1.0",
+    "typechain": "^8.0.0",
+    "typescript": "~4.5.2"
+  },
+  "peerDependencies": {
+    "@ethersproject/abi": "^5.1.2",
```

Comment:
This dependency is technically wrong. The peer dependency version in typechain is `^5.4.7` (see [here](https://github.com/dethcrypto/TypeChain/blob/015abb28bd22826611051f27e0ec96a00f9a0b61/packages/hardhat/package.json#L51)). I used this one to comply with the `check-dependencies` script. What should we do here?

Same thing for the ethers dependency.